src/SFA.DAS.EmployerAccounts.Web/Startup.cs

```diff
@@ -49,57 +50,50 @@ public void Configuration(IAppBuilder app)
 
```

Comment:
Shouldn't app.SetDefaultSignInAsAuthenticationType(CookieAuthenticationDefaults.AuthenticationType); be removed?